app/knowledge_panels.py

```diff
@@ -49,133 +49,47 @@ def data_quality_kp(
     country: Union[str, None] = None,
 ):
     if facet == "country":
-        path = f"{facet}/{value}"
+        country_code = country_to_ISO_code(value=value)
+        country = value
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        path = None
         description = f"{value}"
```

Comment:
same comment as for the PR on last edits. If you want you may try to do as the first kp and unify the facet == "country" case with other cases.